battle-engine.js

```diff
@@ -510,6 +506,16 @@ BattlePokemon = (function () {
 		}
 		return null;
 	};
+	BattlePokemon.prototype.isSuppressing = function (effectType) {
```

Comment:
Air Balloon is not announced either, not even when Magic Room wears off. It can't pop during Magic Room though. Looks like the sim currently handles both of these correctly.